### PR TITLE
[DUOS-1589][risk=no] deprecate-user-rationale

### PR DESCRIPTION
**Pull request by shaemarks**
[Link to Jira ticket](https://broadworkbench.atlassian.net/browse/DUOS-1589)

- removed researcher.rationale from ajax.js 

----
Have you read [Terra's Contributing Guide](https://github.com/DataBiosphere/terra-ui/wiki/Contributor-Guide) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/libs/ajax.js [hunk 1]
```diff
@@ -238,8 +238,6 @@ export const DAR = {
     darInfo.country = rawDar.country;
     darInfo.status = rawDar.status;
     darInfo.restrictions = rawDar.restrictions;
-    darInfo.hasAdminComment = researcher.rationale != null;
```
Comment (rushtong) [inline src/libs/ajax.js:241]: This is used by ApplicationSummary.js:
```
          div({ isRendered: darInfo.hasAdminComment, className: 'row no-margin' }, [
            span({}, [
              label({ className: 'control-label no-padding' }, ['Comments: ']),
              span({ id: 'lbl_adminComment', className: 'response-label', style: { 'paddingLeft': '5px' } },
                [darInfo.adminComment])
            ])
          ]),
```
I suggest that we remove that div altogether.

**Review by rushtong — CHANGES_REQUESTED** [review]
See comment inline, one more deletion should fix it.

**Review by rushtong — APPROVED** [review]
Nice, thank you! 👍🏽
